VideoLocker/src/main/java/org/edx/mobile/view/my_videos/MyRecentVideosFragment.java

```diff
@@ -67,19 +71,22 @@
     @Inject
     protected IEdxEnvironment environment;
 
+    @Inject
+    protected EventBus eventBus;
```

Comment:
I think Ideally, all shared object instances would be provided through dependency injection.